src/games/strategy/triplea/ai/proAI/util/ProUtils.java

```diff
@@ -46,6 +47,26 @@ public ProUtils(final ProAI ai) {
     return unitTerritoryMap;
   }
 
+  public List<PlayerID> getEnemyPlayersInTurnOrder(final PlayerID player) {
+    final GameData data = ai.getGameData();
+    final List<PlayerID> enemyPlayers = new ArrayList<PlayerID>();
+    GameSequence sequence = data.getSequence();
```

Comment:
inconsistent use of optional `final`. Since it's used in the above two lines, it looks intentional, but at the same time it looks like the variables could final, so the inconsistency causes some small confusion.